docs/deployment/caching/rulesets-and-caching-operations.md

```diff
@@ -0,0 +1,300 @@
+---
+myst:
+  html_meta:
+    "description": "Rulesets and caching operations"
+    "property=og:description": "Rulesets and caching operations"
+    "property=og:title": "Rulesets and caching operations"
+    "keywords": "Plone, deployment, automation, caching"
+---
+
+(caching-rulesets-label)=
+
+# Rulesets and caching operations
+
+The caching infrastructure works on the principle of *rulesets* mapped to *caching operations*.
+
+A ruleset is basically just a name, and is normally applied in {term}`ZCML` by the author of a particular view.
+
+There are also some default rulesets applied to general resources - see below.
+
+Please note that `plone.app.caching` places the caching ruleset registry into "explicit" mode. This means that you *must* declare a caching ruleset (with the `<cache:rulesetType />` directive) before you can use it.
+
+Caching operations are components written in Python which either interrupt rendering to provide a cached response (such as a `304 NOT MODIFIED` response), or add caching information to a response (such as setting the `Cache-Control` HTTP response header).
+
+For more details on how to use these components, see the documentation for [plone.caching](https://pypi.org/project/plone.caching).
+
+Once rulesets and caching operations have been registered, they will appear in the caching control panel.
+
+
+## Default rulesets
+
+`plone.app.caching` declares a few default rulesets.  They are listed with descriptions in the control panel.
+
+### Content feed (`plone.content.feed`)
+
+A dynamic feed, e.g. using RSS or ATOM.
+
+### Content files and images (`plone.content.file`)
+
+Includes files and images in content space usually either downloaded or included as an inline element in one of the other public-facing views.
+
+### Content folder view (`plone.content.folderView`)
+
+A public-facing view for a content item that is a folder or container for other items.
+
+### Content item view (`plone.content.itemView`)
+
+A public-facing view for a content item that is not a folder or container for other items.
+
+### File and image resources (`plone.resource`)
+
+Includes images and files created or customised through the ZMI, those exposed in the portal_skins tool, and images registered in resource directories on the filesystem.
+
+### Stable file and image resources (`plone.stableResource`)
+
+Stable resources like the css, and javascript files registered with the Resource Registries.  These are resources which can be cached 'forever'.  Normally this means that if the object does change, its URL changes too.
+
+## Default cache operations
```

Comment:
I don't see any of these default cache operations on the [Plone Classic Demo site](https://classic.demo.plone.org/@@caching-controlpanel#autotoc-item-autotoc-3). Where are they?

Reply:
The demo site has caching installed, but not set up. I now set it up by importing the `Without caching proxy` profile and then enabled caching - then all is there.
(this will be auto-reset soon).

Reply:
@stevepiercy There is a message on top of the page: 
`First time here? We recommend that you get started by importing a preconfigured set of caching rules`

Reply:
OK, but how does a newbie do that? This single sentence does not tell a newbie how to do it, which is why I suggested the above outline.

Reply:
Detailed instructions are missing in `enable.md`, which we could refer to here.

Reply:
I'm fine with wherever it belongs. I need help fleshing out the details of the suggested outline.

Reply:
After (or better before) enable, use the controlpanle to import a profile:
![screenshot_2023-02-25_23:24:02_selection](https://user-images.githubusercontent.com/157140/221382310-8ad49446-b669-467a-902a-af1b794ed06b.png)

Reply:
forgot to say: no need to do more, no advanced/ZMI, nothing else, two steps. I usually import one of the profiles, depending on the setup and then click enable.

Reply:
Resolved in [87dcb50](https://github.com/plone/documentation/pull/1449/commits/87dcb5062e646f50d3486e371d0398bff6529133). Please review. Preview: https://63fb44f1bf95c20008d66c9d--6-docs-plone-org.netlify.app/deployment/caching/rulesets-and-caching-operations.html, then click through to new and existing sections.

Reply:
good, except (see review below) the mention of Squid sneaked in again.